scripts/meta_analysis.py

```diff
@@ -190,74 +191,63 @@ def __lt__(self, other):
                   or (self.chr < other.chr)
                )
 
-    def is_equal(self, other:'VariantData') -> bool:
-        """
-            Checks if this VariantData is the same variant (possibly different strand or ordering of alleles)
-            returns: true if the same false if not
+    def is_equal(self, other: 'VariantData', equalize: bool = False, flip_indels: bool = False) -> bool:
```

Comment:
is equal should not have any effect than telling the answer to this question. thats why they were separated

Reply:
if there is no need for functionality just asking then make this function equalize_to

Reply:
maybe easiest is to introduce private methods. _equalizer() and then public  methods is_equal() and equalize_to()
so to not repeat code

Reply:
Yup, my intention was not repeat code. The private method approach sounds like a plan.

Reply:
Resolved in https://github.com/FINNGEN/META_ANALYSIS/pull/46/commits/c3b166e151ad090fb795de67567017a2e124a818.